src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -53,18 +54,19 @@ export class AboutComponent implements OnInit, OnDestroy {
   }
 
   setVariables() {
+    const NOT_DETECTED = $localize`Not detected`;
```

Comment:
I would rather call it, 'Not Available`. Detected seems to be technical
```suggestion
    const NOT_DETECTED = $localize`Not Available`;
```